service/src/main/resources/templates/ehr_allergy_structure_template.mustache

```diff
@@ -2,6 +2,9 @@
     <CompoundStatement classCode="CATEGORY" moodCode="EVN">
         <id root="{{allergyStructureId}}"/>
         {{{categoryCode}}}
+        {{#confidentialityCode}}
+            {{{confidentialityCode}}}
+        {{/confidentialityCode}}
```

Comment:
I've had a look at the Schema, and the confidentialityCode appears after the availabilityTime. Validating XML with the `confidentialityCode` appearing where you've put it doesn't error, so I'm guessing it's fine.

<details>
  <summary>Schema</summary>

```xml
	<xs:complexType name="RCMR_MT030101UK05.CompoundStatement">
		<xs:sequence>
			<xs:element name="id" type="II" minOccurs="1" maxOccurs="2"/>
			<xs:element name="code" type="CD" minOccurs="1" maxOccurs="1"/>
			<xs:element name="statusCode" type="CS" minOccurs="1" maxOccurs="1"/>
			<xs:element name="effectiveTime" type="IVL_TS" minOccurs="1" maxOccurs="1"/>
			<xs:element name="availabilityTime" type="TS" minOccurs="1" maxOccurs="1"/>
			<xs:element name="priorityCode" type="CV" minOccurs="0" maxOccurs="1"/>
			<xs:element name="confidentialityCode" type="CV" minOccurs="0" maxOccurs="1"/>
			<xs:element name="uncertaintyCode" type="CV" minOccurs="0" maxOccurs="1"/>
			<xs:element name="specimen" type="RCMR_MT030101UK05.Specimen03" minOccurs="0" maxOccurs="unbounded"/>
			<xs:element name="component" type="RCMR_MT030101UK05.Component02" minOccurs="1" maxOccurs="unbounded"/>
			<xs:element name="informant" type="RCMR_MT030101UK05.Informant" minOccurs="0" maxOccurs="unbounded"/>
			<xs:element name="Participant" type="RCMR_MT030101UK05.Participant" minOccurs="0" maxOccurs="unbounded"/>
			<xs:element name="replacementOf" type="RCMR_MT030101UK05.ReplacementOf" minOccurs="0" maxOccurs="unbounded"/>
			<xs:element name="reason" type="RCMR_MT030101UK05.Reason" minOccurs="0" maxOccurs="unbounded"/>
			<xs:element name="reference" type="RCMR_MT030101UK05.Reference" minOccurs="0" maxOccurs="unbounded"/>
			<xs:element name="sequelTo" type="RCMR_MT030101UK05.SequelTo" minOccurs="0" maxOccurs="unbounded"/>
		</xs:sequence>
		<xs:attribute name="type" type="Classes" default="ActHeir"/>
		<xs:attribute name="classCode" type="ActClass" use="optional" default="ACT"/>
		<xs:attribute name="moodCode" type="ActMood" use="optional" default="EVN"/>
		<xs:attribute name="typeID" use="optional">
			<xs:simpleType>
				<xs:list itemType="oid"/>
			</xs:simpleType>
		</xs:attribute>
		<xs:attribute name="realmCode" use="optional">
			<xs:simpleType>
				<xs:list itemType="cs"/>
			</xs:simpleType>
		</xs:attribute>
		<xs:attribute name="nullFlavor" type="cs" use="optional"/>
	</xs:complexType>
```

</details>

Reply:
@adrianclay Yeah, i'd prefer it to be in the right place as per the schema so I'll address this in a new pull request